clarifai/client/dataset.py

```diff
@@ -169,6 +169,12 @@ def list_versions(self, page_no: int = None,
       }
       yield Dataset.from_auth_helper(self.auth_helper, **kwargs)
 
+  def iter_inputs(self):
+    return iter(DatasetExportReader(archive_url=self.archive_zip()))
+
+  def __iter__(self):
+    return self.iter_inputs()
+
```

Comment:
This could be merged into the `__iter__` method:
```python
def __iter__(self):
  return iter(DatasetExportReader(archive_url=self.archive_zip()))
```

Reply:
Modified in this [PR](https://github.com/Clarifai/clarifai-python/pull/315) 👍